RetailCoder.VBE/Inspections/QuickFixes/AssignedByValParameterQuickFix.cs

```diff
@@ -70,75 +75,46 @@ private void SetLocalCopyVariableName()
             }
         }
 
-        private void ModifyBlockToUseLocalCopyVariable()
+        private bool ProposedLocalVariableNameIsValid()
```

Comment:
might be cleaner to just pass the name into this method here. This would allow extracting the validity-check into a separate class, instead of reinventing that wheel over and over across the codebase.